runtime/src/accounts_background_service.rs

```diff
@@ -144,20 +147,27 @@ impl SnapshotRequestHandler {
                 }
                 flush_accounts_cache_time.stop();
 
-                let use_index_hash_calculation = false;
-                let mut hash_time = Measure::start("hash_time");
-                let this_hash = snapshot_root_bank.update_accounts_hash_with_index_option(
-                    use_index_hash_calculation,
-                    test_hash_calculation,
-                    false,
-                );
                 let hash_for_testing = if test_hash_calculation {
+                    let use_index_hash_calculation = false;
+                    let check_hash = false;
+
+                    let (this_hash, _cap) = snapshot_root_bank.accounts().accounts_db.calculate_accounts_hash_helper(
+                        use_index_hash_calculation,
+                        snapshot_root_bank.slot(),
+                        &CalcAccountsHashConfig {
+                            use_bg_thread_pool: true,
+                            check_hash,
+                            ancestors: None,
+                            use_write_cache: false,
+                            epoch_schedule: &EpochSchedule::default(),
+                            rent_collector: &RentCollector::default(),
```

Comment:
To double check, the defaults are the correct values for the epoch schedule and rent collector? I guess we've never changed them then?

Reply:
well, they're not used yet, but they will be. And, they shouldn't be default, so I have fixed that ;-) Good catch! It would have been caught later when we needed these fields.